.github/workflows/publish-iceberg-rest-fixture-docker.yml

```
@@ -42,14 +42,14 @@ jobs:
       run: ./gradlew :iceberg-open-api:shadowJar
     - name: Login to Docker Hub
       run: |
-        docker login -u ${{ secrets.DOCKERHUB_USER }} -p ${{ secrets.DOCKERHUB_TOKEN }}
+        docker login -u apache -p ${{ secrets.DOCKERHUB_TOKEN }}
```

As mentioned in the comment https://github.com/apache/iceberg/pull/11673#issuecomment-2507538087 the user and repository can be different if you're using an organization:
```suggestion
        docker login -u ${{ secrets.DOCKERHUB_USER }} -p ${{ secrets.DOCKERHUB_TOKEN }}
```